// router-library/bundles/router-library.umd.js
(function (global, factory) {
    typeof exports === 'object' && typeof module !== 'undefined' ? factory(exports, require('@angular/core')) :
    typeof define === 'function' && define.amd ? define('router-library', ['exports', '@angular/core'], factory) :
    (global = typeof globalThis !== 'undefined' ? globalThis : global || self, factory(global['router-library'] = {}, global.ng.core));
}(this, (function (exports, i0) { 'use strict';

    var RouterLibraryService = /** @class */ (function () {
        function RouterLibraryService() {
        }
        return RouterLibraryService;
    }());
    RouterLibraryService.ɵprov = i0.ɵɵdefineInjectable({ factory: function RouterLibraryService_Factory() { return new RouterLibraryService(); }, token: RouterLibraryService, providedIn: "root" });
    RouterLibraryService.decorators = [
        { type: i0.Injectable, args: [{
                    providedIn: 'root'
                },] }
    ];
    RouterLibraryService.ctorParameters = function () { return []; };

    var RouterLibraryComponent = /** @class */ (function () {
        function RouterLibraryComponent() {
        }
        RouterLibraryComponent.prototype.ngOnInit = function () {
        };
        return RouterLibraryComponent;
    }());
    RouterLibraryComponent.decorators = [
        { type: i0.Component, args: [{
                    selector: 'lib-router-library',
                    template: "\n    <p>\n      router-library works!\n    </p>\n  "
                },] }
    ];
    RouterLibraryComponent.ctorParameters = function () { return []; };

    var RouterLibraryModule = /** @class */ (function () {
        function RouterLibraryModule() {
        }
        return RouterLibraryModule;
    }());
    RouterLibraryModule.decorators = [
        { type: i0.NgModule, args: [{
                    declarations: [RouterLibraryComponent],
                    imports: [],
                    exports: [RouterLibraryComponent]
                },] }
    ];

    /*
     * Public API Surface of router-library
     */

    /**
     * Generated bundle index. Do not edit.
     */

    exports.RouterLibraryComponent = RouterLibraryComponent;
    exports.RouterLibraryModule = RouterLibraryModule;
    exports.RouterLibraryService = RouterLibraryService;

    Object.defineProperty(exports, '__esModule', { value: true });

})));
