// router-library/fesm2015/router-library.js
import { ɵɵdefineInjectable, Injectable, Component, NgModule } from '@angular/core';

class RouterLibraryService {
    constructor() { }
}
RouterLibraryService.ɵprov = ɵɵdefineInjectable({ factory: function RouterLibraryService_Factory() { return new RouterLibraryService(); }, token: RouterLibraryService, providedIn: "root" });
RouterLibraryService.decorators = [
    { type: Injectable, args: [{
                providedIn: 'root'
            },] }
];
RouterLibraryService.ctorParameters = () => [];

class RouterLibraryComponent {
    constructor() { }
    ngOnInit() {
    }
}
RouterLibraryComponent.decorators = [
    { type: Component, args: [{
                selector: 'lib-router-library',
                template: `
    <p>
      router-library works!
    </p>
  `
            },] }
];
RouterLibraryComponent.ctorParameters = () => [];

class RouterLibraryModule {
}
RouterLibraryModule.decorators = [
    { type: NgModule, args: [{
                declarations: [RouterLibraryComponent],
                imports: [],
                exports: [RouterLibraryComponent]
            },] }
];

/*
 * Public API Surface of router-library
 */

/**
 * Generated bundle index. Do not edit.
 */

export { RouterLibraryComponent, RouterLibraryModule, RouterLibraryService };
